R/drug_parameters.R

```diff
@@ -46,6 +46,7 @@ set_drugs <- function(parameters, drugs) {
 #' @param coverages vector of coverages for this drug
 #' @export
 set_clinical_treatment <- function(parameters, drug, timesteps, coverages) {
+  stopifnot(all(coverages >= 0) & all(coverages <= 1))
```

Comment:
`&` is the bitwise operator while `&&` is the logical operator. Even though `&` works here, it is kind of precarious and doesn't show your intent.